charts/karpenter/templates/webhook/rbac.yaml

```diff
@@ -68,4 +68,10 @@ rules:
   - watch
   - list
   - update
+- apiGroups:
```

Comment:
This role is already applied to the service account via the controller's RBAC. Are you at HEAD? That said, it may not be a bad idea to duplicate here.